src/components/apphost.js

```diff
@@ -167,24 +167,24 @@ define(["appSettings", "browser", "events", "htmlMediaHelper"], function (appSet
         if (browser.mobile) {
             return false;
         }
-
-        var savedResult = appSettings.get(htmlMediaAutoplayAppStorageKey);
-        return "true" === savedResult || "false" !== savedResult && null;
```

Comment:
This function will no longer return any value by default. That could cause issues for anything that calls this expecting a return value.